src/js/App/ErrorComponent/ErrorComponent.js

```diff
@@ -71,14 +72,15 @@ const ErrorComponent = (props) => {
           </Flex>
         </EmptyStateBody>
         <EmptyStatePrimary>
+          {/* Try to reload the page or&nbsp; */}
           <Button
+            component={(props) => (
+              <ChromeLink {...props} appId="landing" href="/">
+                Return to home page
+              </ChromeLink>
+            )}
             variant="primary"
-            onClick={() => {
-              window.history.back();
-            }}
-          >
-            Return to last page.
-          </Button>
+          ></Button>
```

Comment:
There is no need for extra closing tah. You can use just one
```suggestion
          />
```

Reply:
Fixed (tag changed to self-closing)